README: Add first version (#412)
bluerobotics/ping-viewer: README.md

# Ping-Viewer

[![Travis Build Status](https://travis-ci.org/bluerobotics/ping-viewer.svg?branch=master)](https://travis-ci.org/bluerobotics/ping-viewer)
[![Build status](https://ci.appveyor.com/api/projects/status/y4nl4spd3yw1s4bn/branch/master?svg=true)](https://ci.appveyor.com/project/blueroboticsappveyor/ping-viewer/branch/master)
[![Gitter](https://img.shields.io/badge/gitter-online-green.svg)](https://gitter.im/bluerobotics/ardusub/)
![](https://img.shields.io/github/commit-activity/4w/bluerobotics/ping-viewer.svg)
![](https://img.shields.io/github/search/bluerobotics/ping-viewer/TODO:.svg)

<a href="https://bluerobotics.com">
<img src="https://avatars2.githubusercontent.com/u/7120633?v=3&s=200" align="left" hspace="10" vspace="6">
</a>

Graphical user interface for the Blue Robotics Ping echosounder. Ping is a simple, affordable, and compact rangefinder

> for any aquatic project. Ping is [available here](4)!

Reviewer: whoops newline

Reviewer: ```suggestion
for any aquatic project. Ping is [available here][4]!
```

> Graphic user interface for the Ping sonar. Ping is the simple, affordable, and compact ultrasonic altimeter for any aquatic project.

Reviewer: put product link on the bottom as [4], used more than once

The primary goal of _Ping Viewer_ is to be ease to use for both first time and advanced users. It provides full access to the sensor configuration, setup and visualization via [ping-protocol][1].

> All the code is open-source, so you can contribute and evolve it as you want. Follow [Blue Robotics software guidelines][3] for more information.

Reviewer: ```suggestion
All of the code is open-source, so you can contribute and evolve it as you want. Follow [Blue Robotics software guidelines][3] for more information.
```

I think it might be

<br/>

## Quick Start

> Take a look in [our documentation about **installation steps** and **technical information** about ping][2].

Reviewer: ```suggestion
Take a look at [our documentation with **installation steps** and **technical information** about Ping][2].
```

> Ping is [available here](https://www.bluerobotics.com/store/electronics/ping/)!

Reviewer: ```suggestion
```

## Resources

> * [API Reference](1)

Reviewer: ```suggestion
* [API Reference][1]
```

* [Device Specifications](https://www.bluerobotics.com/store/electronics/ping/#tab-technical-details)

> * [Documentation](http://docs.bluerobotics.com)

Reviewer: ```suggestion
* [Application Documentation][2]
```

> * [Forum](5)

Reviewer: ```suggestion
* [Forum][5]
```

> * [API Reference](https://github.com/bluerobotics/ping-protocol)

Reviewer: [1]

> * [Device Specifications](http://www.bluerobotics.com/)

Reviewer: product link [4]

> * [Support](http://docs.bluerobotics.com)

Reviewer: forum
